docs/libcurl/opts/CURLOPT_PROXY_SSLCERT_BLOB.3

```diff
@@ -0,0 +1,69 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2020, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.haxx.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_PROXY_SSLCERT_BLOB 3 "24 Jun 2020" "libcurl 7.71.0" "curl_easy_setopt options"
+.SH NAME
+CURLOPT_PROXY_SSLCERT_BLOB \- set SSL proxy client certificate from memory blob
+.SH SYNOPSIS
+#include <curl/curl.h>
+
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_PROXY_SSLCERT_BLOB, struct curl_blob *stblob);
+.SH DESCRIPTION
+Pass a pointer to a blob structure, which contain information (pointer and
+size) about a memory block with binary data of certificate used to connect
+to the HTTPS proxy. The format must be "P12" on Secure Transport or Schannel.
+The format must be "P12" or "PEM" on OpenSSL. 
+The string "P12" or "PEM" must be specified with \fICURLOPT_PROXY_SSLCERTTYPE(3)\fP.
+
+
+If the blob is initialized with curl_init_blob_dup, the application does not
+have to keep the buffer around after setting this.
+If the blob is initialized with curl_init_blob_persist, the application must
+keep the buffer around after setting this option.
```

Comment:
The`curl_init*` macros were removed, right?

Reply:
removed with new text below